src/ansys/fluent/interface/post_objects/post_object_definitions.py

```diff
@@ -432,15 +432,26 @@ def value(self):
             auto_range_off = self.get_ancestors_by_type(
                 ContourDefn
             ).range.auto_range_off
-            if not filled or (auto_range_off and auto_range_off.clip_to_range()):
-                logger.warning(
-                    "For unfilled and clipped contours node values are displayed."
-                )
-                self._value = True
+            if (
```

Comment:
@prmukherj It feels so natural to write `is_active()` on the spot by factoring out the duplicated code.

Reply:
Done.